compiler/src/main/scala/org/scalajs/nscplugin/PrepJSExports.scala

```diff
@@ -131,13 +111,8 @@ trait PrepJSExports[G <: Global with Singleton] { this: PrepJSInterop[G] =>
    *  constructors are.
    */
   def registerClassOrModuleExports(sym: Symbol): Unit = {
```

Comment:
It is still called from `ScalaJSPlugin#registerModuleExports`. I do not know if anything in the wild actually uses this.

Reply:
[GitHub seems to say](https://github.com/search?l=Scala&q=registerModuleExports&type=Code) that no one uses that method. It's only found in old forks of the scala-js repo itself. So I believe it would be appropriate to get rid of it now. Otherwise it will stay as a completely untested piece of code.